assets/files.wxs

```diff
@@ -1646,124 +1646,124 @@
                 <Directory Id="dir9C2FC7DD1B408AD4F70123DF50131CBD" Name="PackageManagement">
                     <Directory Id="dirBE29553BD60B3A0392916BB9B7487287" Name="1.1.7.0">
                         <Component Id="cmp06B11EAFDD3914ACF2D8C164DC8690DD" Guid="{9A900A1C-06C8-4BA9-A0D0-C8A0B9A19517}">
```

Comment:
Not blocking, but technically you are supposed to change either the component ID or
GUID if you change the file.  Since we shouldn't have to produce a patch, it shouldn't matter.

Reply:
In that case, I'd prefer not to have to change all those ids.

Reply:
Agreed, just mentioning for future reference.